vividus/src/main/java/org/vividus/results/ResultsProvider.java

```diff
@@ -20,14 +20,13 @@
 import java.util.Map;
 import java.util.Optional;
 
-import org.vividus.report.allure.model.Status;
 import org.vividus.results.model.Failure;
 import org.vividus.results.model.NodeType;
 import org.vividus.results.model.Statistic;
 
 public interface ResultsProvider
 {
-    Optional<Status> getRunStatus();
+    int getStatusCode();
```

Comment:
```suggestion
    int calculateExitCode();
```

Reply:
this is exit code in the context of System.exit(...) but as per docs it also called status code that less coupled to exit op